docs/_docs/plugins.md

```diff
@@ -813,7 +813,6 @@ LESS.js files during generation.
 - [Truncate HTML](https://github.com/MattHall/truncatehtml) by [Matt Hall](https://codebeef.com/): A Jekyll filter that truncates HTML while preserving markup structure.
 - [Domain Name Filter by Lawrence Woodman](https://github.com/LawrenceWoodman/domain_name-liquid_filter): Filters the input text so that just the domain name is left.
 - [Summarize Filter by Mathieu Arnold](https://gist.github.com/731597): Remove markup after a `<div id="extended">` tag.
-- [i18n_filter](https://github.com/gacha/gacha.id.lv/blob/master/_plugins/i18n_filter.rb): Liquid filter to use I18n localization.
```

Comment:
Alternatively, we could of course link to https://github.com/gacha/gacha.id.lv/blob/27bdaa0358030d1776d96e4a44f6639b4051f4f1/_plugins/i18n_filter.rb instead.

Reply:
Agreed, we'd better only list gems plugins from now on.